code/modules/admin/verbs/secrets.dm

```diff
@@ -53,14 +53,14 @@
 	switch(action)
 		//Generic Buttons anyone can use.
 		if("admin_log")
-			var/dat = "<B>Admin Log<HR></B>"
+			var/dat = "<meta charset=UTF-8><B>Admin Log<HR></B>"
```

Comment:
Could you wrap the charset in quotes for my sanity though ?

Reply:
It doesn't seem to work this way

Reply:
Done